tlatools/org.lamport.tlatools/test-verify/tlc2/tool/queue/StateQueueJPFTest.java

```diff
@@ -1,29 +1,3 @@
-/*******************************************************************************
- * Copyright (c) 2015 Microsoft Research. All rights reserved. 
```

Comment:
Why did you remove the copyright header?

Reply:
Sorry, It was removed it by mistake I'll add it back!